build/ci/evergreen.yml

```diff
@@ -1366,6 +1364,16 @@ tasks:
     commands:
       - func: "install gotestsum"
       - func: "install podman"
+      - command: shell.exec
+        type: setup
+        params:
+          shell: bash
+          script: |
+            set -e
+            cat /proc/sys/fs/inotify/max_user_watches
+            cat /proc/sys/fs/inotify/max_user_instances
+            echo 256 | sudo tee /proc/sys/fs/inotify/max_user_instances
```

Comment:
it seems most of the flakiness was due to too many open files, bumping this for the test